src/stories/components/custom-popover/custom-popover.scss

```diff
@@ -0,0 +1,4 @@
+.MuiPaper-root {
+  border-radius: 10px !important;
```

Comment:
I don't if is possible not use import, if is necessary is ok, but we should try to avoid to use

Reply:
Solved